api/v1alpha7/types.go

```diff
@@ -292,6 +292,8 @@ type APIServerLoadBalancer struct {
 	AllowedCIDRs []string `json:"allowedCidrs,omitempty"`
 	// Octavia Provider Used to create load balancer
 	Provider string `json:"provider,omitempty"`
+	// Enabled defines whether health monitors should be created for the load balancer.
+	DisabledHealthMonitor bool `json:"disabledHealthMonitor,omitempty"`
```

Comment:
Please could we call the option `DisableHealthMonitors`

Reply:
I think it makes sense